kube/executor.go

```diff
@@ -10,9 +10,27 @@ import (
 	"github.com/c-bata/kube-prompt/internal/debug"
 )
 
+var LivePrefixState struct {
+	LivePrefix string
+	IsEnable   bool
+}
+
+func GetKubeContext() (string) {
+	out, err := exec.Command("/bin/sh", "-c", "kubectl config current-context").Output()
+	if err != nil {
+		fmt.Printf("Got error: %s\n", err.Error())
+	}
+	return strings.TrimRight(string(out), "\n\r") + ">>> "
```

Comment:
would it be possible to add a space before the first `>`? 
its a nitpick so if its too much work don't worry about it at all - i'm really excited to have this

Reply:
np, I've just added leading space [executor.go#L23](https://github.com/MaksymDolgykh/kube-prompt/blob/v1.0.9%2BkubeContextPrompt/kube/executor.go#L23)